sdk/bom/tests.yml

```diff
@@ -0,0 +1,6 @@
+trigger: none
+
+jobs:
```

Comment:
Same here - should this be moved to `eng/pipelines`?

Reply:
Same answer :)